mtags/src/main/scala-2/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -74,6 +73,18 @@ object MultilineStringFormattingProvider {
     if (lastQuote != -1) Some((lastQuote, quoteClosed)) else None
   }
 
+  private def fourQuotes(
```

Comment:
We could have one method like:
```scala
private def hasNQuotes(start: Int, text: String, n : Int): Boolean = {
   (start until n).forall(text(_) == quote)
}
```
and use it also in `isMultilineString`

Reply:
Ok